### PR TITLE
feat: [PIE-3480]: Add cypress test cases for github and gitlab flow

### PR DESCRIPTION
**Pull request by YounixIO**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by YounixIO** [comment]
retrigger titlecheck

**Comment by YounixIO** [comment]
retrigger jest

File: cypress/integration/70-pipeline/PipelineTriggers.spec.ts [hunk 1]
```diff
@@ -161,4 +183,94 @@ describe('Triggers for Pipeline', () => {
...
+    // Configuration Tab
+    cy.fillField('name', 'GitlabTriggerTest')
+    cy.findByText('GitlabTriggerTest').should('exist')
+    cy.get('[value="GitlabTriggerTest"]').should('be.visible')
+
+    cy.contains('span', 'Select Connector').click({ force: true })
+    cy.contains('p', 'gitlab test').click({ force: true })
+    cy.contains('span', 'Apply Selected').click({ force: true })
+
+    cy.get('.custom-event').find('input[placeholder="- Select Event -"]').click({ force: true })
```
Comment (samarth-sharma31) [inline cypress/integration/70-pipeline/PipelineTriggers.spec.ts:258]: Can we query here on the basis of name attribute only --> 
cy.get('input[name="event"]').click({ force: true })

In that case we won't have to add an extra class ("custom-event") in file --> src/modules/72-triggers/pages/triggers/utils/WebhookTriggerConfigPanelUtils.tsx

Comment (YounixIO) [reply cypress/integration/70-pipeline/PipelineTriggers.spec.ts:257]: I tried using the `name` and `data-cy` attributes, but it didn't work in both cases.

**Review by AshwinBhatkal-Harness — DISMISSED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by chiragp1212 — APPROVED** [review]

**Review by samarth-sharma31 — APPROVED** [review]

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 22 Issues 
* Code issue 22 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/8330)

**Pull request closed without merging**